monitor/xfeemngr/tokenprice/buffer.go

```diff
@@ -88,7 +88,7 @@ func (b *Buffer) stream(ctx context.Context) {
 			b.setPrice(token, price)
 		}
 
-		guageBuffered(b.buffer)
+		b.guageBuffered()
```

Comment:
worth fixing the previous typo as well? ie guage -> gauge?

Reply:
https://github.com/omni-network/omni/pull/2052/commits/9df985ffc3434b57aaedea6d74562bddc070ddc7